probe/http/http.go

```diff
@@ -89,13 +90,19 @@ func (h *HTTP) Config(gConf global.ProbeSettings) error {
 		h.Method = "GET"
 	}
 
+	for _, r := range h.SuccessCode {
+		if len(r) != 2 {
+			log.Errorf("HTTP Success Code is invalid - %+v", r)
+			return fmt.Errorf("HTTP Success Code is invalid - %+v", r)
+		}
+	}
+
```

Comment:
if the `SuccessCode` is empty, we can add the default range [0-499] into the SuccessCode. Then you don't need the if-else in `DoProbe()` function.